src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -707,6 +707,9 @@ public void Initialize()
             {
                 NotificationCenter.Add(alertRaised);
             }
+            
+            if(PlatformDetails.RunningOnDocker == false)
```

Comment:
minor: space after `if`

Reply:
Shouldn't we check this on Linux?

Reply:
I am not sure I follow your question.
The check here is because first I check it on docker on Linux and I got the host value of Swap so it is not relevant unless we will implement the `MemoryInformation.GetMemoryInfo` differently .

Reply:
Did you docker instance had limited memory?
What about:
https://docs.microsoft.com/en-us/dotnet/api/system.gcmemoryinfo.totalavailablememorybytes?view=netcore-3.1

Reply:
The physical memory is correct but the swap is incorrect (as I saw )

Reply:
@haludi I got the feeling that this is relevant only for Linux. But it's not. It verifies the page file size on windows as well - is that correct?

Reply:
Did you see my question?

Reply:
> @haludi I got the feeling that this is relevant only for Linux. But it's not. It verifies the page file size on windows as well - is that correct?

My check? yes.
Do you think it should be applied only on Linux?

Reply:
If your check verifies the size of a page file on windows then it's okay. Although then the hint message shouldn't mention about "swap" but "the page file"